message_sent.py

```diff
@@ -0,0 +1,48 @@
+import pymsteams
+import requests
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Teams_Send_Message_Logger")
+
+
+class MessageException(Exception):
+    pass
+
+
+class WebhookUrlError(MessageException):
+    pass
+
+
+def prepare_content_string(quote_text, quote_author, image_url):
+    logger.info('Preparing content string...')
+    content_string = f"{quote_text}\n\n" \

```

Comment:
Try to cover it in "()" in order to get rid of a "\" characters

Reply:
As i understand, in case of f-strings we can't get rid of "", just can change them into '' characters.
Or i can put it all in one string, but i think it would be less readable.

Reply:
Ou, I understand, that moreover i can add '()' to avoid using '\'. Do you think it's better to do in this way?

Reply:
Just slash isn't seen in comments))))